app/src/main/res/values-sw600dp-land/dimens.xml

```diff
@@ -399,4 +399,7 @@
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
 
+  <!-- ProfileListFragment -->
+  <dimen name="profile_list_fragment_recycler_view_padding_bottom">40dp</dimen>
+
 </resources>
```

Comment:
As the `profile_list_fragment_recycler_view_padding_bottom` for both sw600dp and mobile layout is same (40dp) therefore we don't need to create another dimens here.